test/integration/clinicians/clinician-sidebar.js

```diff
@@ -17,38 +17,28 @@ const groups = [
   },
 ];
 
+const testClinician = {
```

Comment:
I moved this up to prevent duplication, but this same named `const` was used with different data in the test further down, so I renamed it.